test/common/upstream/health_check_fuzz.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package test.common.upstream;
+
+import "validate/validate.proto";
+import "test/fuzz/common.proto";
+import "envoy/config/core/v3/health_check.proto";
+
+message Respond {
```

Comment:
suggest that Respond contains an HttpResponse/TcpResponse/GrpcResponse? That way other actions aren't duplicated for the different health checking types.

Reply:
Great idea! I think this is the best way to go!